### PR TITLE
Add intro about MSTest

### PR DESCRIPTION
**Pull request by Evangelink**
---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/testing/unit-testing-mstest-intro.md](https://github.com/dotnet/docs/blob/1859d82a8970b02143116a6409ec0d138ae5d52c/docs/core/testing/unit-testing-mstest-intro.md) | [MSTest overview](https://review.learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-mstest-intro?branch=pr-en-us-40045) |

### CODE REVIEW

**Review by MarcoRossignoli — APPROVED** [review]
good

**Review by BillWagner — COMMENTED** [review]
I had one grammar / sentence structure suggestion @Evangelink 

Then, this is ready to merge.

File: docs/core/testing/unit-testing-mstest-intro.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+title: MSTest overview
+description: Learn about MSTest.
+author: Evangelink
+ms.author: amauryleve
+ms.date: 03/17/2024
+---
+
+# MSTest overview
+
+MSTest, Microsoft Testing Framework, is a test framework for .NET applications. It allows to write and execute tests, tests suites with integration to Visual Studio and Visual Studio Code Test Explorers, .NET CLI, and many CI.
```
Comment (Evangelink) [inline docs/core/testing/unit-testing-mstest-intro.md:11]:
```suggestion
MSTest, Microsoft Testing Framework, is a test framework for .NET applications. It allows to write and execute tests, tests suites with integration to Visual Studio and Visual Studio Code Test Explorers, .NET CLI, and many CI pipelines.
```

**Comment by Evangelink** [comment]
Thanks, I have applied the change @BillWagner

File: docs/core/testing/unit-testing-mstest-intro.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+MSTest, Microsoft Testing Framework, is a test framework for .NET applications. It allows to write and execute tests, tests suites with integration to Visual Studio and Visual Studio Code Test Explorers, .NET CLI, and many CI pipelines.
+
+This is a fully supported, open-source and cross-platform test framwork that works with all supported .NET Frameworks (.NET Framwork, .NET Core, .NET, UWP, WinUI...) hosted on [GitHub](https://github.com/microsoft/testfx).
+
+## MSTest support policy
+
+Since v3.0.0, MSTest is strictly following [semantic versioning](../../csharp/versioning.md#semantic-versioning).
+
+MSTest team only supports the latest released version and strongly encourages its users and customers to always update to latest version to benefit from new improvements and security patches. Preview releases aren't supported by Microsoft but they are offered for public testing ahead of the final release.
```
Comment (IEvangelist) [inline docs/core/testing/unit-testing-mstest-intro.md:19]:
```suggestion
The MSTest team only supports the latest released version and strongly encourages its users and customers to always update to latest version to benefit from new improvements and security patches. Preview releases aren't supported by Microsoft, but they are offered for public testing ahead of the final release.
```

File: docs/core/testing/unit-testing-mstest-intro.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+title: MSTest overview
+description: Learn about MSTest.
+author: Evangelink
+ms.author: amauryleve
+ms.date: 03/17/2024
+---
+
+# MSTest overview
+
+MSTest, Microsoft Testing Framework, is a test framework for .NET applications. It allows to write and execute tests, tests suites with integration to Visual Studio and Visual Studio Code Test Explorers, .NET CLI, and many CI pipelines.
```
Comment (IEvangelist) [inline docs/core/testing/unit-testing-mstest-intro.md:11]:
```suggestion
MSTest, Microsoft Testing Framework, is a test framework for .NET applications. It allows you to write and execute tests, and provide test suites with integration to Visual Studio and Visual Studio Code Test Explorers, the .NET CLI, and many CI pipelines.
```

File: docs/core/testing/unit-testing-mstest-intro.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+This is a fully supported, open-source and cross-platform test framwork that works with all supported .NET Frameworks (.NET Framwork, .NET Core, .NET, UWP, WinUI...) hosted on [GitHub](https://github.com/microsoft/testfx).
```
Comment (IEvangelist) [inline docs/core/testing/unit-testing-mstest-intro.md:13]:
```suggestion
MSTest is a fully supported, open-source and a cross-platform test framework that works with all supported .NET targets (.NET Framework, .NET Core, .NET, UWP, WinUI, and so on) hosted on [GitHub](https://github.com/microsoft/testfx).
```

**Review by IEvangelist — APPROVED** [review]

**Pull request merged**